extensions/f1-standings/src/driver.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { ActionPanel, Action, Icon, List, Color } from "@raycast/api";
+import axios from "axios";
+
+interface DriverStanding {
+  position: number;
+  points: number;
+  wins: number;
+  Driver: {
+    driverId: string;
+    permanentNumber: string;
+    code: string;
+    url: string;
+    givenName: string;
+    familyName: string;
+    dateOfBirth: string;
+    nationality: string;
+  };
+  Constructors: {
+    constructorId: string;
+    url: string;
+    name: string;
+    nationality: string;
+  }[];
+}
+
+interface State {
+  season?: number;
+  items?: DriverStanding[];
+  error?: Error;
+}
+
+export default function Command() {
+  const [state, setState] = useState<State>({});
+  useEffect(() => {
+    async function fetchDrivers() {
+      try {
+        const res = await axios.get("https://ergast.com/api/f1/current/driverStandings.json");
+        const data = await res.data;
+        setState({
+          items: data.MRData.StandingsTable.StandingsLists[0].DriverStandings,
+          season: data.MRData.StandingsTable.season,
+        });
+      } catch (error) {
+        console.log(error);
+        setState({
+          error: error instanceof Error ? error : new Error("Something went wrong"),
+        });
+      }
+    }
+
+    fetchDrivers();
+  }, []);
+
+  return (
+    <List isLoading={!state.items && !state.error}>
+      <List.Section title={String(state.season) ?? ""}>
+        {state.items?.map((item) => (
+          <List.Item
+            key={item.Driver.driverId}
+            icon={{
+              source: item.position + ".png",
```

Comment:
While the visual part is kinda cool, the font looks nice, but I'd just use the default font to keep things simple.

Reply:
But for the icon in the list I need something of type:
```Typescript
ImageLike: URL | Asset | Icon | FileIcon | Image;
```
Also when just prefixing the driver/team name with the position as Text the names left alignment "jump" around which looks really bad. Or do you mean the font within the image should be replaced?